addons/reloadlaunchers/functions/fnc_reloadLauncher.sqf

```diff
@@ -1,13 +1,15 @@
 #include "script_component.hpp"
 /*
- * Author: commy2
+ * Author: commy2, johnb43
  * Reload a launcher
+ * If the ammo argument is nil, a full magazine will be given.
  *
  * Arguments:
  * 0: Unit to do the reloading <OBJECT>
  * 1: Target to rload <OBJECT>
```

Comment:
```suggestion
 * 1: Target to reload <OBJECT>
```

Reply:
I addressed that in #9334, but I can merge it here.

Reply:
no it's fine